py/torch_tensorrt/dynamo/fx_ts_compat/lower.py

```diff
@@ -67,13 +71,17 @@ def compile(
         timing_cache_prefix: Timing cache file name for timing cache used by fx2trt.
         save_timing_cache: Update timing cache with current timing cache data if set to True.
         cuda_graph_batch_size: Cuda graph batch size, default to be -1.
-        use_experimental_fx_rt: Uses the next generation TRTModule which supports both Python and TorchScript based execution (including in C++).
+        explicit_batch_dimension: Whether to specify an explicit batch dimension to TRT
+        use_experimental_rt: Uses the next generation TRTModule which supports both Python and TorchScript based execution (including in C++).
+        max_aux_streams: max number of aux stream to use
+        version_compatible: enable version compatible feature
+        optimization_level: builder optimization level
     Returns:
         A torch.nn.Module lowered by TensorRT.
     """
-    if use_experimental_fx_rt and not explicit_batch_dimension:
+    if use_experimental_rt and not explicit_batch_dimension:
```

Comment:
Will this work since use_experimental_fx_rt was removed?

Reply:
`use_experimental_fx_rt ` was renamed to `use_experimental_rt` in the function header, for more consistent naming across the board (every other function uses `use_experimental_rt`).

Reply:
I've since removed that change, since I noticed `torch_tensorrt.fx.compile(...)` still uses `use_experimental_fx_rt`, so it could be better to keep those the same